### PR TITLE
Fix job

### PR DESCRIPTION
**Pull request by mcbenjemaa**

### CODE REVIEW

File: helm/prometheus-meta-operator/templates/alertmanager/ownership-job/job.yaml [hunk 1]
```diff
@@ -40,6 +40,8 @@ spec:
               namespace=$(kubectl get chart --namespace=giantswarm -l app.kubernetes.io/name=prometheus-meta-operator --output=jsonpath="{.items[0].spec.namespace}")
               kubectl patch --type=merge --record=false -p '{"metadata":{"annotations":{"meta.helm.sh/release-name": "'"${name}"'","meta.helm.sh/release-namespace": "'"${namespace}"'"},"labels":{"app.kubernetes.io/managed-by": "Helm"}}}' -n "${namespace}" alertmanager alertmanager
               managedByLabel=$(kubectl get alertmanager -n "${namespace}" alertmanager -o=jsonpath='{.spec.alertmanagerConfigSelector.matchLabels.app\.kubernetes\.io/managed-by}')
-              [ "$managedByLabel" == "prometheus-meta-operator" ] && kubectl patch --record=false -n "${namespace}" --type json --patch '[{ "op": "remove", "path": "/spec/alertmanagerConfigSelector/matchLabels/app.kubernetes.io~1managed-by", "value": "prometheus-meta-operator" }]' alertmanager alertmanager
+              if [[ "$managedByLabel" "$managedByLabel" == "prometheus-meta-operator" ]]; then
```
Comment (marieroque) [inline helm/prometheus-meta-operator/templates/alertmanager/ownership-job/job.yaml:43]: twice "$managedByLabel" is it normal ?

Comment (mcbenjemaa) [reply helm/prometheus-meta-operator/templates/alertmanager/ownership-job/job.yaml:43]: Thanks a lot )

Comment (mcbenjemaa) [reply helm/prometheus-meta-operator/templates/alertmanager/ownership-job/job.yaml]: fixed

**Comment by mcbenjemaa** [comment]
@marieroque don't leave before you approve 😄

**Comment by mcbenjemaa** [comment]
I'm testing

**Review by marieroque — APPROVED** [review]

**Comment by mcbenjemaa** [comment]
tested

**Pull request merged**